packages/components/src/components/as-category-widget/as-category-widget.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../styles/src/core/_dev';
+
+$bar--height: get-spacing(1);
+
+as-category-widget {
+  --category-widget--description--color: #{$color-type-20};
+  --category-widget--bar--height: #{$bar--height};
+  --category-widget--background-color: #{$color-ui-10};
+  --category-bar--background-color: #{$color-ui-20};
+
+  display: block;
+  background: var(--category-widget--background-color, $color-ui-10);
+
+  @include padding(4);
+
+  .as-category-widget__description,
+  .as-category-widget__count {
+    color: var(--description--color, $color-type-20);
+    color: $color-type-20;
+  }
+
+  .as-category-widget__list {
+    @include padding(0 0 4);
+  }
+
+  .as-category-widget__description + .as-category-widget__list {
+    margin-top: get-spacing(4);
+  }
+
+  .as-category-widget__category-title {
+    position: relative;
+
+    &::after {
+      content: attr(data-value);
+      position: absolute;
+      right: 0;
+    }
+  }
+
+  .as-category-widget__bar {
+    position: relative;
+    width: 100%;
+    height: $bar--height;
+    border-radius: get-spacing(0.5);
+    background-color: var(--category-bar--background-color, $color-ui-20);
+    background-color: $color-ui-20;
+  }
+
+  .as-category-widget__value {
```

Comment:
What do you think if we avoid nesting the classes? i know it's more organized ... but I'd like a clearer css output

Reply:
Nesting inside `as-category-widget`? I don't really mind putting them outside `as-category-widget`. We're using BEM so there shouldn't be any collisions :)

What do you think @IagoLast @rubenmoya @ivanmalagon?